packages/@lightningjs/ui-components/src/components/Base/Base.d.ts

```diff
@@ -19,18 +19,41 @@
 import lng from '@lightningjs/core';
 import { SpeechType } from '../../mixins/withAnnouncer';
 
-export default class Base extends lng.Component {
-  skipPlinko: boolean;
-  centerInParent: boolean;
-  _announce: SpeechType;
-  _whenEnabled: Promise<void>;
-  _getFocusScale(): number;
-  _getUnfocusScale(): number;
-  _update(): void;
-  _focus(): void;
-  _unfocus(): void;
-  _smooth?: boolean;
-
-  set announce(announce: SpeechType);
-  get announce(): SpeechType;
+declare namespace Base {
+  export interface TemplateSpec extends lng.Component.TemplateSpec {}
+
+  export interface EventMap extends lng.Component.EventMap {}
+
+  export interface TypeConfig extends lng.Component.TypeConfig {
+    skipPlinko: boolean;
+    centerInParent: boolean;
```

Comment:
TypeConfig shouldn't really contain any of this stuff. None of it should actually do anything. TypeConfigs are for when you want to modify the internal component types of `EventMapType`, `SignalMapType`, `TextureType`, `IsPage` and `HistoryStateType`

https://lightningjs.io/docs/#/lightning-core-reference/TypeScript/Components/TypeConfigs

Reply:
Because the `EventMap` on line 25 is not included in the TypeConfig here, it's not being used in any way either.